setup.py

```diff
@@ -161,7 +161,7 @@ def main():
             "-DBUILD_DOCS=OFF",
             "-DPYTHON3_LIMITED_API=ON",
             "-DBUILD_OPENEXR=ON",
-            "-DBUILD_PNG=ON",
+            "-DBUILD_ZLIB=ON",
```

Comment:
The idea was to update `zlib` also, because we have the old version in the system (2020 year release), but there is the new version also, which can be built from source.